controllers/api/index.js

```diff
@@ -1,7 +1,6 @@
 const router = require('express').Router();
-
 const userRoutes = require('./userRoutes');
 
-router.use('/users', userRoutes);
+router.use('/user', userRoutes);
```

Comment:
does public/js/login.js need to be updated to match the change in the route name

Reply:
fixed to /users